zio-http/src/main/scala/zhttp/service/server/content/handlers/ServerResponseHandler.scala

```diff
@@ -14,26 +14,15 @@ import java.io.RandomAccessFile
 
 @Sharable
 private[zhttp] trait ServerResponseHandler[R] {
```

Comment:
Create a followup issue to move this into a service and inject into `Handler.scala`

Reply:
done https://github.com/dream11/zio-http/issues/1085

Reply:
PR https://github.com/dream11/zio-http/pull/1086